### PR TITLE
feat(surveys): UI for surveys

### PR DESCRIPTION
**Pull request by liyiy**
## Problem

UI for surveys

Use the feature flag `surveys` to play around with just the UI of it. Some logic has been set up but it's not connected to a working backend yet so you can just use fake data for now.

## Changes

Screen Shot 2023-05-24 at 1 07 54 AM
Screen Shot 2023-05-24 at 1 07 44 AM
Screen Shot 2023-05-24 at 1 08 35 AM

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

**Comment by liyiy** [comment]
@neilkakkar This is probably a good place to start on just the UI so you don't have to review an omega frontend + backend PR 😅 

Definitely not the final UI version, will keep iterating, once I get the backend sorted and connected.

File: frontend/src/scenes/surveys/surveyLogic.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+const DEFAULT_DATATABLE_QUERY: DataTableNode = {
+    kind: NodeKind.DataTableNode,
+    full: true,
+    source: {
+        kind: NodeKind.EventsQuery,
+        select: ['*', 'event', 'timestamp', 'person'],
+        orderBy: ['timestamp DESC'],
+        after: '-30d',
+        limit: 100,
+        event: 'insight viewed',
```
Comment (neilkakkar) [inline frontend/src/scenes/surveys/surveyLogic.tsx:38]: reminder to change this in a follow up^

File: frontend/src/scenes/surveys/surveyLogic.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+                    } else {
+                        console.error('Invalid query', query)
+                        return DEFAULT_DATATABLE_QUERY
+                    }
+                },
+            },
+        ],
+    }),
+    selectors({
+        isSurveyRunning: [
```
Comment (neilkakkar) [inline frontend/src/scenes/surveys/surveyLogic.tsx:105]: reminder to add breadcrumbs

**Review by neilkakkar — APPROVED** [review]
Looks fine to me!

Makes sense to iterate over the UI, haven't left any comments on it for now, just two non-obvious things for a follow up.

Also take extra care not to release the flag to anyone else in this stage :)

File: frontend/src/scenes/surveys/surveyLogic.tsx (hunk 1 above)
Comment (liyiy) [reply frontend/src/scenes/surveys/surveyLogic.tsx:38]: yesssir

**Pull request merged**